data/categories.txt

```diff
@@ -1860,7 +1860,7 @@ uk:3^Метро|транспорт
 vi:3^Ngầm
 zh-Hans:^地铁|地铁|地铁|地铁|交通
 zh-Hant:1^地鐵|捷運|2地下鐵|運輸
-el:^Υπόγειος|σωλήνας|μετρό|υπόγεια|μεταφορές
+el:^Μετρό|υπόγειος|ηλεκτρικός|σιδηρόδρομος|μεταφορές|συγκοινωνίες
```

Comment:
The "συγκοινωνίες" will be added from the ```@transport``` group category, please fix
it there if "συγκοινωνίες" is better than "Συγκοινωνία" (or leave both synonyms there).

Reply:
ping on this comment